konfera/event/views.py

```diff
@@ -127,3 +128,13 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
```

Comment:
I just realized that we select `event` and `order` individually so user can change the `slug` and see an invalid order for another event. 

I suggest to remove the `slug` from the URL (and get the event object via `order.ticket.type.event`). (hmm.. but there is a problem!)

@ricco386 I think `Order` should have a `ForeignKey` to `Ticket`, currently it's vice versa, which IMO doesn't make sense.